central/src/main/webapp/components/ModelNavigator.jsx

```diff
@@ -86,25 +98,117 @@ export default function ModelNavigator(props) {
 	const modelZooData = useFetch(URL);
 
 	const [model, setModel] = useState(null);
+	const [modelList, setModelList] = useState([]);
```

Comment:
Change the modelLists default value from [] into modelZooData. That way, users start in the page having models to look at before being able to search to help find the model they want.